packages/components/package.json

```diff
@@ -6,7 +6,8 @@
   "dependencies": {
     "@kiwicom/margarita-utils": "^0",
     "@kiwicom/universal-components": "0.0.11",
-    "react-native-shimmer-placeholder": "https://github.com/tomzaku/react-native-shimmer-placeholder.git#expo"
+    "react-native-shimmer-placeholder": "https://github.com/tomzaku/react-native-shimmer-placeholder.git#expo",
+    "shortid": "^2.2.14"
```

Comment:
Why do we need this? Would there be a way to do without this dependency?

Reply:
The current use case of this is just for the key in `map`.
And we should not generate the Id on GraphQL, because it later can lead to misunderstanding.

Reply:
We should generate some ID field for this on the server. We should not use shortID. It beats the purpose of the [key](https://reactjs.org/docs/lists-and-keys.html#keys) prop.

Reply:
So, ID is now sector index which is get from GraphQL, is it OK?

Reply:
Anything that uniquely identifies the sector is useable